guestlib/extensions/command_batch_worker.h

```diff
@@ -0,0 +1,34 @@
+#ifndef _AVA_GUESTLIB_COMMAND_WORKER_H_
+#define _AVA_GUESTLIB_COMMAND_WORKER_H_
+
+#include <glib.h>
+
+#include "common/cmd_channel.hpp"
+#include "common/support/thread.h"
+#include "guestlib/guest_thread.h"
+#include "guestlib/queue_worker.h"
+
+namespace ava {
+
+class CmdBatchingWorker : public QueueWorker {
```

Comment:
`Worker` is a confusing naming, as the API server is also called "worker". Maybe `GeustCmdBatchingQueue: public CmdQueue`?